src/warnet/lnchannel.py

```diff
@@ -100,22 +82,54 @@ def compare_attributes(attr1, attr2, min_value=0, attr_name=""):
                 self.logger.debug(f"Mismatch in {attr_name}: {attr1} != {attr2}")
             return result
 
+        if self.node1_policy is None:
+            return ch2.node1_policy is None
+
+        if self.node2_policy is None:
+            return ch2.node2_policy is None
+
+        if ch2.node1_policy is None:
+            return self.node1_policy is None
+
+        if ch2.node2_policy is None:
+            return self.node2_policy is None
```

Comment:
I think its too early to return here?
example, if `ch 1 policy 1 == None and ch 2 policy 1 == None` we'd return True before comparing policy 2?

Reply:
Excellent catch. Reworked logic.